### PR TITLE
plot map returns grid data besides basemap

### PR DESCRIPTION
**Pull request by forero**
This minor change allows `grid_data` manipulation to make and compare histograms of different maps.

### CODE REVIEW

**Comment by weaverba137** [comment]
@geordie666, please verify that this fixes #68.

**Comment by geordie666** [comment]
I believe that this will fix #68 when merged, yes. However, I'd recommend at least the following under review:

1. The documentation (https://github.com/desihub/desiutil/blob/15b74974621cdfdd3531557e539b616872951150/py/desiutil/plots.py#L885-L889) needs changed to include `grid_data` as a return

2. As the addition of an extra returned array will potentially break a lot of downstream code, I'd be tempted to add an input `return_grid_data=False` that will only return the `grid_data` if prompted, e.g.:

```
if return_grid_data is True:
    return basemap, grid_data

return basemap
```
That's just personal taste, though. People could also just update their calls to `plot_sky_binned` as and when they break.

**Comment by weaverba137** [comment]
Please add a brief note about this to the `changes.rst` file before merging.

**Comment by moustakas** [comment]
I agree with @geordie666 that returning `grid_data` should be optional, not the default, although I *can* deal with code breakage if that's inconvenient.

**Comment by sbailey** [comment]
I'd like to make new tags today, so I made the changes suggested by @geordie666 (make new grid_data return optional) and @weaverba137 (update changes.rst) and will merge after Travis tests pass again.

File: doc/changes.rst [hunk 1]
```diff
@@ -13,11 +13,14 @@ Change Log
   incorrectly special-cased the north pole with brick sizes that don't
   evenly divide 180 degrees (PR `#84`_).
 * Added tests of :mod:`desiutil.sklearn` (PR `#86`_).
+* Adds ``return_grid_data``` option to ``desiutil.plots.plot_sky_binned()``
```
Comment (weaverba137) [inline doc/changes.rst]: I think there's a couple of problems here.  This line should be:
```
* Adds ``retrun_grid_data`` option to :func:`desiutil.plots.plot_sky_binned`.
```

**Comment by dkirkby** [comment]
I won't have time to test this today, but it sounds fine to me as long as the default return type has not changed (since I am using the original API in many places).

**Pull request merged**